tests/js/testUtils.js

```diff
@@ -56,4 +56,12 @@ https://github.com/gpii/universal/LICENSE.txt
         });
     };
 
+    gpii.tests.utils.simulateKeyEvent = function (onElement, keyEvent, keyCode) {
```

Comment:
@jobara, these two functions are serving different purposes. What about adding comments to explain what they do?